### PR TITLE
Create scripts to launch benchmark testing to Google cloud

### PR DESCRIPTION
**Pull request by dimberman**
# Description
## What is the current behavior?
Currently users are running benchmark tests on their local machines. This is a) inconsistent as much of the time might be tied to network time and b) bad as it discourages consistently running benchmarks (E.g. as a daily job or as a part of CI).

closes: https://github.com/astronomer/astro-sdk/issues/432

## What is the new behavior?

Now users will have the ability to launch a GKE cluster, push an image for the astro SDK, and launch the job to run on GKE

## Does this introduce a breaking change?
No

### Checklist
- [ ] Created tests which fail without the change (if possible)
- [ ] Extended the README / documentation, if necessary
- [ ] Be able to run the benchmark, with different configurations, in the cloud (GCP and GKE)

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -14,7 +14,7 @@ license = {file = "LICENSE"}
 
 requires-python = ">=3.7"
 dependencies = [
-    "apache-airflow>=2.0",
+    "apache-airflow>=2.2",
```
Comment (kaxil) [inline pyproject.toml:17]: Why do we need this?

File: tests/benchmark/config.json [hunk 2]
```diff
@@ -5,64 +5,17 @@
...
-      "name": "ten_kb",
-      "path": "gs://astro-sdk/benchmark/trimmed/covid_overview/covid_overview_10kb.parquet",
-      "rows": 385817,
-      "size": "10 KB"
-    },
-    {
-      "conn_id": "bigquery",
-      "file_type": "csv",
-      "name": "hundred_kb",
-      "path": "gs://astro-sdk/benchmark/trimmed/tate_britain/artist_data_100kb.csv",
```
Comment (kaxil) [inline tests/benchmark/config.json:27]: Why is all this removed?

**Review by kaxil — COMMENTED** [review]
Did the script run fine on GKE now?

**Comment by dimberman** [comment]
@kaxil I can confirm that this runs smoothly on GKE. I was able to do the entire flow from beginning to end. Glad to demo tomorrow if you like.

File: tests/benchmark/config.json (hunk 2 above)
Comment (dimberman) [reply tests/benchmark/config.json]: Added missing test cases. I think this was more about facilitating quick testing.

File: pyproject.toml (hunk 1 above)
Comment (dimberman) [reply pyproject.toml]: Fixed

**Comment by dimberman** [comment]
@kaxil I'm gonna run this script one more time and merge if everything works.

File: tests/benchmark/config.json [hunk 3]
```diff
@@ -37,32 +36,9 @@
       "size": "9.9M"
     },
     {
-      "conn_id": "bigquery",
-      "file_type": "ndjson",
-      "name": "one_gb",
-      "path": "gs://astro-sdk/benchmark/trimmed/stackoverflow/stackoverflow_posts_1g.ndjson",
-      "rows": 385817,
-      "size": "1G"
```
Comment (kaxil) [inline tests/benchmark/config.json:45]: Why remove this from here?

Comment (dimberman) [reply tests/benchmark/config.json]: @kaxil I'm not sure hwo that happened. I added it back in.

**Pull request merged**